proto/metadata.proto

```diff
@@ -38,6 +38,7 @@ message Metadata {
     TESTBED_DUT_DUT_ATE_2LINKS = 6;
     TESTBED_DUT_ATE_8LINKS = 7;
     TESTBED_DUT_400ZR = 8;
+    TESTBED_DUT_400ZR_PLUS = 9;
```

Comment:
Also need to add testbed file here: https://github.com/openconfig/featureprofiles/tree/main/topologies